R/github-actions.R

```diff
@@ -1,114 +1,76 @@
-# general GHA setup ------------------------------------------------------------
-
-#' GitHub Actions setup
+#' Set up a GitHub Action
```

Comment:
```suggestion
#' Set up a GitHub Actions workflow
```

More inline with GitHub's verbiage and works nicely with the `Workflows` heading below.